docs/java.md

```diff
@@ -0,0 +1,373 @@
+  <groupId>com.scylladb</groupId>
+  <artifactId>java-driver-core</artifactId>
+  <version>3.11.2.0</version>
+</dependency>
+
+<dependency>
+  <groupId>com.scylladb</groupId>
+  <artifactId>java-driver-query-builder</artifactId>
+  <version>3.11.2.0</version>
+</dependency>
+
+<dependency>
+  <groupId>com.scylladb</groupId>
+  <artifactId>java-driver-mapper-runtime</artifactId>
+  <version>3.11.2.0</version>
+</dependency>
+```
+
+## 2. Connecting to the Cluster
+
+Get your database credentials from your [ScyllaDB Cloud Dashboard](https://cloud.scylladb.com/clusters) in the tab `Connect`.
+
+```java
+import com.datastax.driver.core.Cluster;  
+import com.datastax.driver.core.PlainTextAuthProvider;  
+import com.datastax.driver.core.Session;  
+import com.datastax.driver.core.policies.DCAwareRoundRobinPolicy;  
+  
+class Main {  
+  
+    public static void main(String[] args) {  
+    Cluster cluster = Cluster.builder()  
+        .addContactPoints("your-node-url.scylla.cloud", "your-node-url.clusters.scylla.cloud", "your-node-url.clusters.scylla.cloud")  
+        .withLoadBalancingPolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("AWS_US_EAST_1").build()) // your local data center  
+        .withAuthProvider(new PlainTextAuthProvider("scylla", "7H8VbxJTuG2flYK"))  
+        .build();  
+    
+    Session session = cluster.connect();  
+    
+    }  
+}
+```
+
+> If the connection gets refused, check if your IP Address is added to the list of allowed IP addresses.
+
+## 3. Handling Queries
+
+With the Java driver, you can use the function inside your cluster connection called `execute(query)` and build the query you want to execute inside your database/keyspace.
+
+```java
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.PlainTextAuthProvider;
+import com.datastax.driver.core.policies.DCAwareRoundRobinPolicy;
+
+class Main {  
+  
+    public static void main(String[] args) {  
+        Cluster cluster = Cluster.builder()  
+            .addContactPoints("your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud")  
+            .withLoadBalancingPolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("AWS_US_EAST_1").build()) // your local data center  
+            .withAuthProvider(new PlainTextAuthProvider("scylla", "*******"))  
+            .build();  
+        
+        Session session = cluster.connect();  
+        ResultSet result = session.execute("SELECT * FROM system.clients LIMIT 10");  
+  
+        System.out.println(result);
+    }
+}
+```
+
+
+### 3.1 Creating a Keyspace
+
+The `keyspace` inside the ScyllaDB ecosystem can be interpreted as your `database` or `collection`.
+
+On your connection boot, you don't need to provide it but you will use it later and also is able to create when you need.
```

Comment:
This sentence is not clear. I'm adding a suggestion below, but I'm not sure this is what you want to say. Anyway, this sentence requires rewording.

`You don't need a keyspace on your connection boot, but you'll need it to create a table.`

Reply:
It was exactly what I meant. The idea is to explain that you don't need a keyspace/database to use it. 

Thanks for the rephrasing!